Cargo.toml

```diff
@@ -138,8 +138,8 @@ optional = true
 features = ["serde"]
 
 [dependencies.mongocrypt]
-git = "https://github.com/mongodb/libmongocrypt-rust.git"
-branch = "main"
+git = "https://github.com/abr-egn/libmongocrypt-rust.git"
```

Comment:
I'll revert this once https://github.com/mongodb/libmongocrypt-rust/pull/47 goes in.